app/controllers/mfa_controller.rb

```diff
@@ -3,6 +3,7 @@
 require 'auth/authentication_backend'
 
 class MfaController < ApplicationController
+  include ControllerConcern
   include AuthenticationBackend
   include ERB::Util
```

Comment:
Can we remove this include?

Reply:
We need the ConctrollerConcern include to provide QR Codes for the MFA controller..  So this one has to stay.

Reply:
the MFA controller should no longer be used...